src/main/java/edu/greenblitz/tobyDetermined/Robot.java

```diff
@@ -112,13 +112,16 @@ public void robotPeriodic() {
 	@Override
 	public void disabledInit() {
 		CommandScheduler.getInstance().cancelAll();
+
+		BreakCoastSwitch.getInstance().setIsDisabled(true);
```

Comment:
Consider changing the name to something clearer, such as
```suggestion
		BreakCoastSwitch.getInstance().setDisabled(true);
```

In any case, couldn't you just get this from the driver station?